src/Page/Food.elm

```diff
@@ -510,8 +543,54 @@ type alias UpdateIngredientConfig =
     }
 
 
+createConfig : Query.IngredientQuery -> UpdateIngredientConfig -> BaseElement.Config Ingredient Mass Msg
+createConfig ingredientQuery { excluded, db, recipeIngredient, impact, selectedImpact } =
```

Comment:
Nit: I realize this now lives at the root of `Page/Food.elm`, so maybe we should name it more explicitely eg. `createElementSelectorConfig`